packages/template/base/src/BaseTemplate.ts

```diff
@@ -47,12 +47,17 @@ export class BaseTemplate implements ForgeTemplate {
           }
           const srcFiles = ['index.css', 'index.js', 'index.html', 'preload.js'];
 
-          for (const file of rootFiles) {
-            await this.copy(path.resolve(tmplDir, file), path.resolve(directory, file.replace(/^_/, '.')));
-          }
-          for (const file of srcFiles) {
-            await this.copy(path.resolve(tmplDir, file), path.resolve(directory, 'src', file));
-          }
+          await Promise.all(
+            rootFiles.map(async (file) => {
+              this.copy(path.resolve(tmplDir, file), path.resolve(directory, file.replace(/^_/, '.')));
+            })
+          );
+
+          await Promise.all(
+            srcFiles.map(async (file) => {
+              this.copy(path.resolve(tmplDir, file), path.resolve(directory, 'src', file));
+            })
+          );
```

Comment:
Reduce one loop and improve the readability of the code.
```suggestion
          await Promise.all(
            rootFiles.map(async (file) => {
              const sourcePath = path.resolve(tmplDir, file);
              await this.copy(sourcePath, path.resolve(directory, file.replace(/^_/, '.')));
              await this.copy(sourcePath, path.resolve(directory, 'src', file));
            })
          );
```

Or

```suggestion
          await Promise.all(
            rootFiles.map(async (file) => {
              const sourcePath = path.resolve(tmplDir, file);
              
              return Promise.all([
                this.copy(sourcePath, path.resolve(directory, file.replace(/^_/, '.'))),
                this.copy(sourcePath, path.resolve(directory, 'src', file))
              ]);
            });
          );
```

Or


```suggestion
          await Promise.all(
            rootFiles.map((file) => {
              const sourcePath = path.resolve(tmplDir, file);
              
              return [
                this.copy(sourcePath, path.resolve(directory, file.replace(/^_/, '.'))),
                this.copy(sourcePath, path.resolve(directory, 'src', file))
              ];
            }).flat();
          );
```

> When there is only one await in Promise.all, we prefer to leave it to the Promise itself. If there is more than one, we use await in the Promise's function to improve the readability of the code.

Reply:
We aren't doing both of these things to root files though- depending if it's a root or source file, we copy the files into their respective correct path. 

I can concat the arrays together, and replace the second argument in `this.copy()` to a variable, but I don't think that's any more readable than what is there already